### DISCUSSION TITLE
System doesnt allow for duplicate filters

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15528] [Feature improvement]
I checked it on stable and gitdev.

If admin user create filter (public or not) the other user(s) cannot create filter with the same name. 
I consider it as a bug, as this will be very common conflicting situation in larger systems (with many users).

Best!

**Comment by mariuszkrzaczkowski** [comment]
Using duplicate names will be very misleading.

**Reply by diderich** [reply]
What about the case:
User A creates a private filter "My Contacts".
User B wants to create a private filter "My Contacts"
This may be a common scenario, but will fail.
I would suggest/prefer having only the combination "filter name"+"filer creator/owner" unique!?

**Reply by mariuszkrzaczkowski** [reply]
Let's discuss it with other members and see what they think

**Reply by JohnMaverick** [reply]
Duplicate record, filter already exists

System I am setting now may have hundreds of users. Each can create own set of filters. Duplicated name bringing lots of confusion for users. Lets allow them to have the same name of filter. Maris say it will be missleading, but... each user can see only own filter. Maybe do like this: User can see only own filter name, Admin can see Filter name + username (or ID)
Hard to force admin to deal with users filters (sharing) from settings panel - it will be time waster

**Comment by garryatbrighton** [comment]
Indeed, filters with the same name are impossible. When you have 50 users, there will be great chance that some of them want to use the same filter name.
